modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2631,6 +2643,25 @@ TEST_P(Test_ONNX_layers, Conformance_Gemm_transposeB) {
     testONNXModels("test_gemm_transposeB", pb, 0, 0, false, true, 2);
 }
 
+TEST_P(Test_ONNX_layers, Conformance_Expand_dim_changed) {
```

Comment:
> Conformance

What does it mean here?

We should not duplicate tests with `test_onnx_conformance.cpp`

Reply:
They are not duplicates. The very original form of them cannot be supported by the current dnn engine, because they have dynamic input shape. I think this could be some kind of a waste that we cannot test them just because we cannot handle dynamic input shape. So I modified the model, set the dynamic input shape as constant with corresponding tensors.

Also see comment above https://github.com/opencv/opencv/pull/24295#issuecomment-1728965356. The script for model modification is in the pr comment in opencv extra.

Reply:
"Conformance_" is removed from test names.